### PR TITLE
Style and Finishing Touches

### PR DESCRIPTION
**Pull request by Feruzteame**
closes #14 
oh My God, It was hard work to style more than 400 line. by the way i try my best. may be i change some thing in your style. but in some part i change totally the html structure.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -184,4 +186,237 @@ header button{
...
         margin-left: 0;
     }
- }
\ No newline at end of file
+ }
+
+
+ /*Guessing game*/
+ 
+html, .heroPicture {
```
Comment (abelRoland) [inline style.css:194]: Here was `html, body` but I believe that was giving conflicts with the other pages. For this reason that I have done another css file and didn't connect the page that I did to the main css file. But it's ok, I believe that we just need to delete the `.heroPicture` from this line.

**Review by abelRoland — COMMENTED** [review]
Hey @Feruzteame thanks for the work, it was really a lot.
There is a problem with style in the game that I did now, when the result appears, normally each information appears in a different line, now they appear all in one line. But we try to fix this after.

Greetings

**Comment by Feruzteame** [comment]
Really ! Oh sorry, so what can we do ?

**Comment by Feruzteame** [comment]
Do u think it is easy to do

**Pull request merged**